lessons/11-basic-data-structs/assignments/account-generator.md

```diff
@@ -0,0 +1,28 @@
+# Student Account Generator
+
+We are going to write a program that generates some student information. This exercise will practice our skills with strings, random numbers, arrays and iterators.
+
+## Requirements
+
+1. You will start by creating three arrays:
+    - An array that will contain student names
+    - An array that will store student ID numbers
+    - An array that will contain student email addresses    
+
+2. Write a `times` loop that asks the user to enter in 5 student names (first and last name) and store those values in the first array in **all uppercase**.
+
+3. Write a `times` loop to generate random student ID numbers from 111111 to 999999 and add these values to the student ID number array.
```

Comment:
change _add_ to _put_ just to make sure they don't think they are doing 11235 + 20371 or something weird.